applications/nrf_desktop/description.rst

```diff
@@ -1131,6 +1131,11 @@ Building information
 Use the configuration with the ``release`` file suffix for the HID report rate measurement.
 Debug features, such as logging or assertions, decrease the application performance.
 
+.. note::
```

Comment:
The related PR is already merged. The notes could be seen here-  https://github.com/nrfconnect/sdk-nrf/blob/v3.2-branch/applications/nrf_desktop/description.rst?plain=1#L1135. Not sure why git is not showing the conflict. The note could be removed. Related PR - https://github.com/nrfconnect/sdk-nrf/pull/26100